crates/subspace-farmer/src/plot.rs

```diff
@@ -598,6 +621,35 @@ impl IndexHashToOffsetDB {
             .wrapping_add(&PieceDistance::MIDDLE)
     }
 
+    fn get_piece_index_hash(&self, distance: PieceDistance) -> PieceIndexHash {
```

Comment:
This and above methods become confusing. It doesn't really get anything from anywhere. Those are conversion methods. Something like `piece_distance_to_hash` and `piece_hash_to_distance` would be a better symmetric names for those.